### PR TITLE
FT2-664 Cache reload implemented. 

### PR DESCRIPTION
**Pull request by ravi-methods**

### CODE REVIEW

File: DFC.App.Banners.Data/Models/CmsApiModels/PageBannerContentItemApiDataModel.cs [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
 
+using Newtonsoft.Json;
+
 namespace DFC.App.Banners.Data.Models.CmsApiModels
 {
     [ExcludeFromCodeCoverage]
     public class PageBannerContentItemApiDataModel : BaseContentItemModel
     {
-        // TODO: add properties.
+        [JsonProperty("PageUrl")]
```
Comment (pramireddy) [inline DFC.App.Banners.Data/Models/CmsApiModels/PageBannerContentItemApiDataModel.cs]: Please add the following properties
        [JsonProperty(PropertyName = "banner_WebPageURL")]
        public string? PageLocation { get; set; }

        [JsonProperty(PropertyName = "banner_WebPageName")]
        public string? WebPageName { get; set; }

**Review by pramireddy — COMMENTED** [review]
Code changes are looks fine.

File: DFC.App.Banners.Data/Models/CmsApiModels/PageBannerContentItemApiDataModel.cs (hunk 1 above)
Comment (ravi-methods) [reply DFC.App.Banners.Data/Models/CmsApiModels/PageBannerContentItemApiDataModel.cs]: That's an outdated file.

**Review by pramireddy — APPROVED** [review]
It's looks good.

**Pull request merged**